src/seedu/addressbook/AddressBook.java

```diff
@@ -200,7 +200,18 @@
      */
     public static void main(String[] args) {
         showWelcomeMessage();
-        processProgramArgs(args);
+        if (args.length >= 2) {
```

Comment:
think about other constructs such as variables that also contribute towards abstraction.